interpreter/cling/lib/Interpreter/InterpreterCallbacks.cpp

```diff
@@ -187,11 +201,23 @@ namespace cling {
     }
 
     if (enableDeserializationListenerCallbacks && Reader) {
-      // FIXME: need to create a multiplexer if a DeserializationListener is
-      // alreday present.
+      // Create a new deserialization listener.
       m_DeserializationListener.
         reset(new InterpreterDeserializationListener(this));
-      Reader->setDeserializationListener(m_DeserializationListener.get());
+
+      // Wrap the deserialization listener in an MultiplexConsumer and then
+      // combine it with the existing Consumer.
+      // FIXME: Maybe it's better to make MultiplexASTDeserializationListener
+      // public instead. See also: https://reviews.llvm.org/D37475
+      std::unique_ptr<DeserializationListenerWrapper> wrapper(new DeserializationListenerWrapper(m_DeserializationListener.get()));
```

Comment:
Does that fit into cling's / clang's 80 char line width limit?

Reply:
Nope, that's why Travis is red :) Fixed!